### ISSUE TITLE
Includes/Excludes property seems to not work

### ISSUE THREAD

**Issue by centic9** [schmutterer/gradle-openjpa#3]
I tried to use includes/excludes to make the openjpa only enhance the relevant classes, however it seems to have no effect whatsoever even though I tried various different ways to specify the properties.

BTW, the current description in the README is definitely incorrect, there is no trailing single-quote and Gradle wants me to use an equal sign.

```
openjpa {
    excludes ['**/notthese/*]
}
```
should likely be

```
openjpa {
    excludes = ['**/notthese/*']
}
```

Can you please update the example to also include a working includes-property? I at least tried the following, but none of them worked

in an openjpa-tag:
```
includes = ['**/jpa/*']
includes = ['**/jpa/**']
includes = '**/jpa/*'
...
```

also 
```
openjpa.includes = ...
```
no success there either...

**Comment by ChristophGr** [comment]
This seems to be a bug in openjpa.

I couldn't get it to work in maven either (and it invokes the enhancer quite similar there).

I'll remove this capability from the documentation

**Comment by ChristophGr** [comment]
OK, scratch that.
I changed the implementation on master (and corrected the documentation)

It's now
```
openjpa {
    files = fileTree(sourceSets.main.output.classesDir).matching {
        exclude '**/Unwanted.class'
    }
}
```

**Comment by robert-nemet** [comment]
openjpa {
    files = fileTree(sourceSets.main.output.classesDir).matching {
        excludes = ['**/exception/*', '**/facade/*', '**/integration/*', '**/service/*']
    }
}

this works with 0.2.0..

We just had to add this to build script:

buildscript{
...
dependencies {
        classpath 'org.apache.openjpa:openjpa:2.3.0'
        classpath 'at.schmutterer.oss.gradle:gradle-openjpa:0.2.0-SNAPSHOT'
    }
...
}

**Comment by ChristophGr** [comment]
OK, the solution is not ideal (as it cannot be easily passed on to subprojects), but it's included in 0.2.0 which should be available on central soon.

**Comment by centic9** [comment]
Please note that the syntax seems to be "excludes = ", but the README.md states "exclude ", i.e. "s" and "=" are missing.

It might also be useful to list the syntax where multiple can be specified, e.g. 

```
    files = fileTree(sourceSets.main.output.classesDir).matching {
        excludes = ['**/action/**', '**/excel/**', '**/util/**', '**/test/*.class']
    }    
```
